src/AspNet.Security.OpenId.Steam/SteamAuthenticationOptions.cs

```diff
@@ -12,6 +12,14 @@ public SteamAuthenticationOptions()
     {
         Authority = new Uri(SteamAuthenticationDefaults.Authority);
         CallbackPath = SteamAuthenticationDefaults.CallbackPath;
+        Backchannel = new HttpClient
+        {
+            DefaultRequestHeaders =
+            {
+                { "referer", "https://steamcommunity.com" },
+                { "origin", "https://steamcommunity.com" }
+            }
+        };
```

Comment:
I don't think we'd want to ship this if it's just a temporary workaround.

You can apply it directly to the options instance when you configure things:

```csharp
builder.AddSteam(options =>
{
    options.Backchannel = new HttpClient
    {
        DefaultRequestHeaders =
        {
            { "referer", "https://steamcommunity.com" },
            { "origin", "https://steamcommunity.com" }
        }
    };
});
```

Reply:
Agreed, let's wait before making any change. As @martincostello said, it's easy to work around locally.